packages/clangml/clangml.4.1.0/opam

```diff
@@ -0,0 +1,35 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "OCaml bindings for Clang API"
+description: """
+clangml provides bindings to call the Clang API from OCaml.
+"""
+maintainer: ["Thierry Martinez <thierry.martinez@inria.fr>"]
+authors: ["Thierry Martinez <thierry.martinez@inria.fr>"]
+license: "BSD-2-Clause"
+homepage: "https://memcad.gitlabpages.inria.fr/clangml/"
+doc: "https://memcad.gitlabpages.inria.fr/clangml/doc/clangml/index.html"
+bug-reports: "https://gitlab.inria.fr/memcad/clangml/issues"
+depends: [
+  "conf-libclang"
+  "conf-ncurses"
+  "conf-zlib"
+  "dune" {>= "1.10.0"}
```

Comment:
According to the dune-project file, you are using lang 1.11.
Can you update the bound also upstream?

Reply:
Fixed in https://github.com/ocaml/opam-repository/pull/16446/commits/5d0e453624df25718384b54691295e442158be4c (and upstream: https://gitlab.inria.fr/memcad/clangml/-/commit/003002d33d2fe55f7dd80a9ad24470018eb7e6d8 )
Thanks!